src/views/homepage/components/featured-content-grid/featured-content-grid.module.css

```diff
@@ -0,0 +1,58 @@
+.root {
+	color: var(--token-color-foreground-high-contrast);
+	display: grid;
+	gap: 60px;
+	grid-template-areas:
+		'b a a'
+		'b c d';
```

Comment:
💭 Might be nice to semantically name these so its easy to read the page layout from this file.

Reply:
Do you have some names already in mind? I'll have to noodle on this a little bit!

Reply:
Ah yes I see now that the cards could probably be filled with any content so might not make sense to name them. I'll resolve this!